components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -2106,6 +2107,18 @@ private IdentityProvider getIDP(Connection dbConnection, String idPName, int idp
                 federatedIdp.setFederatedAuthenticatorConfigs(getFederatedAuthenticatorConfigs(
                         dbConnection, idPName, federatedIdp, tenantId));
 
+                // Get IdP secrets from IDN_SECRET table.
```

Comment:
Improve the comment to imply that the secrets are retrieved from the db and decrypted

Reply:
Resolved in [7f035fe](https://github.com/wso2/carbon-identity-framework/pull/4430/commits/7f035fedca5ed28a27f590804ea2f47e1042da4d)